docs/scripts/plugin_documentation

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+function plugin_doc {
+	plugin=$1
+	dest=$2
+	product=`echo $plugin | sed -e 's/-.*//'`
+	if [ $product = "foglamp" ]; then
+		org=dianomic
+	else
+		org=fledge-iot
+	fi
+	type=`echo $plugin | sed -e 's/fledge-//' -e 's/foglamp-//' -e 's/-.*//'`
+	name=`echo $plugin | sed -e 's/fledge-//' -e 's/foglamp-//' -e "s/${type}-//"`
+	mkdir -p /tmp/doc.$$
+	(
+		cd /tmp/doc.$$
+		git clone https://github.com/${org}/${plugin}.git
```

Comment:
Instead of cloning all the repositories and checking for doc directory, can we have list of repositories which then will be cloned only? Or with time we are going to have docs in all repositories?

Reply:
We will have docs in all repositories over time. I really want it to be dynamic so we don't have to remember to add new repositories.